docker-compose/docker-compose-htcmock/docker-compose.yml

```diff
@@ -24,6 +24,7 @@ services:
       - Amqp__MaxPriority=10
       - Amqp__User=admin
       - Amqp__Password=admin
+      - Amqp__MaxRetry=5
```

Comment:
Couldn't you add this configuration in the appsettings? this would allow to have a default value and make the current infrastructure compatible with this change.

Reply:
Yes, I plan to do so